apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/NotesDialog.tsx

```diff
@@ -19,7 +19,7 @@ interface NotesDialogProps {
   notes: Maybe<string> | undefined;
 }
 
-const PoolCandidate_UpdateMutation = graphql(/* GraphQL */ `
+const PoolCandidate_NotesMutation = graphql(/* GraphQL */ `
   mutation UpdatePoolCandidate(
```

Comment:
We need to change this line as well, this was the one causing the issue.

```suggestion
  mutation PoolCandidate_UpdateNotes(
```

Reply:
Fixed in [fix mutation name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/69bacd9173e73754f0e2be849d98e89094a9636a)

Once we get translations we can merge 👍

Reply:
I think that was mistake 😆 thank for the fix